web/book/src/syntax/expressions-and-operators.md

```diff
@@ -15,21 +15,91 @@ filter circumference > 10 && color != "red"
 
 ## Operator precedence
 
-This table gives formal description of operator precedence. Because function
-calls have the lowest precedence, nested function calls or arguments that start
-or end with an operator require parentheses.
+This table shows operator precedence. Use parentheses `()` to prioritize
+operations and for function calls (see the discussion below.)
 
 <!-- markdownlint-disable MD033 — the `|` characters need to be escaped, and surrounded with tags rather than backticks   -->
 
-| Group          | Operators         | Precedence | Associativity |
-| -------------- | ----------------- | ---------- | ------------- |
-| identifier dot | `.`               | 1          |               |
-| unary          | `- + ! ==`        | 2          |               |
-| range          | `..`              | 3          |               |
-| mul            | `* / %`           | 4          | left-to-right |
-| add            | `+ -`             | 5          | left-to-right |
-| compare        | `== != <= >= < >` | 6          | left-to-right |
-| coalesce       | `??`              | 7          | left-to-right |
-| and            | `&&`              | 8          | left-to-right |
-| or             | <code>\|\|</code> | 9          | left-to-right |
-| function call  |                   | 10         |               |
+|          Group | Operators         | Precedence | Associativity |
+| -------------: | ----------------- | :--------: | :-----------: |
+|    parentheses | `()`              |     0      |   see below   |
```

Comment:
@aljazerzen for your post-merge review. Not really an operator but I thought it did belong in the precedence table, let us know if not

Reply:
I replied about that somewhere - it's fine even if they are technically not operators. , understandable enough.